scene/animation/animation_tree.cpp

```diff
@@ -540,6 +540,11 @@ bool AnimationTree::_update_caches(AnimationPlayer *player) {
 	List<StringName> sname;
 	player->get_animation_list(&sname);
 
+	Ref<Animation> reset_anim;
+	bool has_reset_anim = player->has_animation("RESET");
```

Comment:
Reset is not ensured to exist and even if it does, the rest poses are not ensured to exist either, so I am not sure this is a good solution.

Reply:
@reduz If we don't have ResetPose or RestPose, we will have the problem of going through Quaternion(0,0,0;1) only for rotation, but on the other hand, if we have ResetPose, we can blend without penetrating the body, as in the video above, so this feature may not need to be the default, but it needs to be there somewhere.